### DISCUSSION TITLE
Overview page template component usage

### DISCUSSION THREAD

**Discussion by jweintraub** [twilio-labs/paste#1403] [Office hours] [answered]
Adding a discussion here in preparation for office hours:

Would like to review the Overview page template and the usage of components 

Background: This is a templated page that will be used for every product in Console to allow users to evaluate the products
Stage of design: High-fidelity
Research: [synthesis & notes](https://docs.google.com/spreadsheets/d/18lgGzuqAGsOtKViWCLnV0L3GYu5ymXFoQ4rW1lqcq8Q/edit?usp=sharing)
Links to mockups or prototype: [Prototype](https://www.figma.com/file/1RoWCc0EF90GfQWIbDlJo8/Overview-page?node-id=146%3A26984)

Thank you!!

**Comment by nkrantz** [comment]
Thanks for sharing these links, @jweintraub. We'll take a look and prepare some thoughts for Office Hours next week!

**Comment by TheSisb** [comment, marked as answer]
Following up on today's the Office Hours. [Here's the link to the annotated recording](https://www.loom.com/share/accfd0ac8e554554a1b0154a6ca01988) (password is in slack). Thanks for coming!
